src/Base/Placement.cpp

```diff
@@ -221,8 +221,13 @@ std::string Placement::toString() const
     double angle {};
     rot.getRawValue(axis, angle);
 
-    // clang-format off
-    return fmt::format("position ({:.1f}, {:.1f}, {:.1f}), axis ({:.1f}, {:.1f}, {:.1f}), angle {:.1f}\n",
-                       pos.x, pos.y, pos.z, axis.x, axis.y, axis.z, angle);
-    // clang-format on
+    return fmt::format(
+        "position ({:.1f}, {:.1f}, {:.1f}), axis ({:.1f}, {:.1f}, {:.1f}), angle {:.1f}\n",
+        pos.x,
+        pos.y,
+        pos.z,
+        axis.x,
+        axis.y,
+        axis.z,
+        angle);

```

Comment:
Why is this change part of this PR? It seems @wwmayer very specifically disabled the clang-format here, so I'm inclined to respect that.

Reply:
These are the annoying stuff wwmayer refers to in https://github.com/FreeCAD/FreeCAD/pull/18606#issuecomment-2553348605
There's been a lot of these issues  in PRs lately, but a simple rebase on main has fixed it on the others. @luzpaz, did you change rebase the branch when force pushing?

Reply:
Here is a rebased branch without the changes in Placement.cpp: https://github.com/FreeCAD/FreeCAD/pull/18646

Reply:
> change rebase the branch when force pushing  

How does one 'change rebase' ? Sorry, i thought i was doing what wwmayer asked. I have a rudimentary understanding of git, and apologize for that.

Reply:
> How does one 'change rebase' ?

git checkout main
git pull
git checkout typos2-dec-2024 # this must be your local branch with only commit. If not then do a `git reset --hard HEAD~1` to remove the pre-commit. But double check it with  `git log -2`
git rebase main
git push --force luzpaz typos2-dec-2024

Reply:
> If not then do a `git reset --hard HEAD~1` to remove the pre-commit. But double check it with  `git log -2`

oh ok..  good to know since I assumed that just by simply resetting my branch and removing the pre-commit commit and resubmitting it that it would stop triggering the action each time. TIL